### DISCUSSION TITLE
Installing latest version from ISO

### DISCUSSION THREAD

**Discussion by Tanver-Hasan** [Security-Onion-Solutions/securityonion#5042] [Unsupported Versions]
When isntalling latest version of security onion using ISO image in Virtual Box , the installation fails. I looked into the logs and see following error message. I used import mode. 

<img width="725" alt="Screenshot 2021-08-02 at 18 44 30" src="https://user-images.githubusercontent.com/38843176/128022461-4485cfd0-9116-4123-8354-a8d630ba5db9.png">

In terms of Network, I have enabled both NAT and Bridge Adapter. Then I tried NAT and Host Only Adapter

**Comment by InfosecGoon** [comment]
It looks like the issue may be with DNS - it's not resolving the host repo,securityonion.net, so it can't pull down the software to install.
